app/controllers/hq/orphans_controller.rb

```diff
@@ -3,4 +3,63 @@ class Hq::OrphansController < HqController
   def show
     @orphan = Orphan.find(params[:id])
```

Comment:
Oh, i got it. Cool. But yes, we need to know that this is happening, because can we difficult to follow the flow of the code or track issues. I'm more tempt to use it without `before_action`. Good observation tho :+1: